src/app/pages/users/tags/tags-list-table-data-source.ts

```diff
@@ -314,6 +318,15 @@ export class TagsListTableDataSource extends TableDataSource<Tag> {
           (actionDef as TableImportTagsActionDef).action(ImportDialogComponent, this.dialog);
         }
         break;
+      case UserButtonAction.EXPORT_TAGS:
+        if (actionDef.action) {
+          const filterValues = this.buildFilterValues();
+          filterValues['WithUser'] = 'true';
```

Comment:
I don't think this filtering has to be done here.
Filtering of the list should be reused.
If extra params need to be forced then add them in the backend.

Reply:
I forgot to ask, will we let the choice to the user (checkbox or whatever) to chose if he wants to export with users or not ?

Reply:
For the time being let's always export all the tags with all the users and we'll see if the customers request this feature.